src/drivers/mongodb.ts

```diff
@@ -56,17 +56,43 @@ export default defineDriver((opts: MongoDbOptions) => {
       const document = await getMongoCollection().findOne({ key });
       return document?.value ?? null;
     },
+    async getItems(items) {
+      const keys = items.map((item) => item.key);
+
+      const result = await getMongoCollection()
+        .find({ key: { $in: keys } })
```

Comment:
Would it make sense if we use aggregation `$match` => `$getFields`? (might be faster if we let the server side keep result sorted)

Reply:
Agreegate operations preserve order unlike find + `$in`.

I don't know why wrote getField yes project 😆

Reply:
But would it not then preserve the order of how every the server stored it, so order of insertion and not order of user keys?

So this:

```js
  const result = await getMongoCollection()
    .aggregate([
      { $match: { key: { $in: keys } } },
      { $project: { key: 1, value: 1 } },
    ])
    .toArray();
```

Would not work.

Or am I still missing something?